src/Microsoft.DocAsCode.Build.TableOfContents/BuildTocDocumentStepBase.cs

```diff
@@ -87,7 +87,7 @@ private void BuildCore(TocItemViewModel item, FileModel model, IHostService host
 
             void UpdateDependencies(HashSet<string> linkTos, Dictionary<string, ImmutableList<LinkSourceInfo>> linkSources, string link)
             {
-                var path = UriUtility.GetPath(link);
+                var path = HttpUtility.UrlDecode(UriUtility.GetPath(link));
```

Comment:
see https://github.com/dotnet/docfx/pull/4246

`HtmlUtility.UrlDecode` turns `+` into ` `

Reply:
Use `HttpUtility.UrlDecode` can keep consistent with all other parts in v2. Users can use `%2B` to work around this, or we can change it to `Uri.UnescapeDataString` throughout v2 later.